atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -323,6 +330,17 @@ def set_group_manifest_info(self, extra, worker_metadatas):
                             instance['extra']['docker']['repositories'] = repositories
                             self.log.debug("reset tags to so that docker is %s",
                                            instance['extra']['docker'])
+                            annotations = get_worker_build_info(self.workflow, platform).\
```

Comment:
In the near future, we should consider breaking up `set_group_manifest_info` into smaller methods. As it stands it's quite difficult to follow along.

Reply:
Agree, we should run cyclomatic complexity checker (`mccabe` from PyPi) and split large functions into smaller ones. I don't think its worth doing in this PR though